src/XLParser/ExcelFormulaGrammar.cs

```diff
@@ -207,13 +215,13 @@ public ExcelFormulaGrammar() : base(false)
             #region Functions
 
             FunctionCall.Rule =
-                  Function + Arguments + CloseParen
+                  FunctionName + Arguments + CloseParen
                 | PrefixOp + Formula
                 | Formula + PostfixOp
                 | Formula + InfixOp + Formula
                 ;
-
-            Function.Rule = ExcelFunction | UDFToken;
+                
+            FunctionName.Rule = ExcelFunction;
```

Comment:
We can consider removing the FunctionName rule and use the ExcelFunction directly